server/src/email/assets/html-template.es6

```diff
@@ -1,101 +1,35 @@
 'use strict';
 
-const template = {};
-
-template.forestService = `
-    <style>
-      .header {
-        background-color: #eeece5;
-        padding: 1em;
-      }
-      
-      .body {
-        color: #212121;
-        font-family: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif;
-        padding: 1em 0;
-      }
-      
-      a { 
-        color: #076727;
-      }
-      
-      h1 {
-        color: #076727;
-      }
-
-      h2 {
-        font-size: 2rem;
-        font-weight: 700;
-      }
-
-      h3 {
-        font-size: 1.5rem;
-        font-weight: 700;
-      }
-
-      h1, h2, h3, h4, h5, h6 {
-        font-family: "Merriweather", "Georgia", "Cambria", "Times New Roman", "Times", serif;
-        clear: both;
-        line-height: 1.3;
-        margin-bottom: .5em;
-        margin-top: 1em;
-      }
+const htmlTemplateStyle = require('./html-template-style.es6');
 
-      img {
-        max-width: 100%;
-      }
-
-      table, 
-      th, 
-      td {
-       border: 0;
-      }
-
-      table.bordered {
-        width: 100%;
-        border: 1px solid #dfe2e5;
-      }
-      
-      .bordered td, 
-      .bordered th {
-        font-weight: 400;
-        padding: 1em;
-      }
-
-      .border-bottom {
-        border-bottom: 1px solid #dfe2e5;
-      }
-      
-      .border-right {
-        border-right: 1px solid #dfe2e5;
-      }
-      
-      hr {
-        background-color: #656565;
-        border: none;
-        height: 1px;
-        margin: 30px 0;
-      }
+const template = {};
 
-      ul {
-        padding-left: 1em;
-        margin-bottom: 2em;
-      }
-      
-      .logo {
-        max-height: 100px;
-      }
-      
-    </style>
-    <div class="header">
+template.forestService = (content, title) => {
+  return `
+    ${htmlTemplateStyle}
```

Comment:
This is nice, separating these things out.  I bet it makes it a lot easier to find what you're looking for.